src/Estranged.Lfs.Adapter.S3/S3BlobAdapter.cs

```diff
@@ -62,12 +64,8 @@ public Task<SignedBlob> UriForUpload(string oid, long size, CancellationToken to
         {
             return Task.FromResult(new SignedBlob
             {
-                Uri = MakePreSignedUrl(oid, HttpVerb.PUT, BlobConstants.UploadMimeType),
-                Expiry = config.Expiry,
-                Headers = new Dictionary<string, string>
-                {
-                    {"Content-Type", BlobConstants.UploadMimeType}
-                }

```

Comment:
The S3 supports headers, if the API in use does not support headers, it's not S3 compatible

Reply:
It does support headers, but it's `UploadMimeType` that breaks it. 
So I was asking if we could remove it if not necessary for AWS.

It it's mandatory, would you like for me to parameterize the content type?

This small PR would open to many cloud providers which are compatible with S3. I'm sure the community will understand you give support mostly for AWS, and they'll contribute if other cloud providers bring problems (like I did).

Reply:
You said you couldn't test it, but maybe you could test it on AWS S3. 
It does not make sense in real use as the default config would suffice, but it can assure you it works.
🙃

Reply:
Another use case that could be useful for people with this. It allows to use an AWS S3 bucket that's outside the stack, even in another region 😉

Reply:
Hello @alanedwardes, have you been able to try testing as I suggested?